### PR TITLE
Update README.md - version release

### PR DESCRIPTION
**Pull request by mikiw**
## Development related changes

- Added description how to handle version release

## Checklist:

- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Documented the changes
- [x] Linked the issues which this PR resolves

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -675,3 +675,42 @@ You don't need to build anything to be able to run locally, but if you need the
 ```text
 poetry build
 ```
+
+### Development - Version release
+
+You can check the current version on master with these commands:
+```text
```
Comment (FabijanC) [inline README.md:682]: you can remove the `text` language directive

File: README.md [hunk 2]
```diff
@@ -675,3 +675,42 @@ You don't need to build anything to be able to run locally, but if you need the
...
+git checkout master
+poetry version
+```
+
+To update the version use:
```
Comment (FabijanC) [inline README.md:687]: Perhaps replace L687-L694 with a text directing to this link: https://python-poetry.org/docs/cli/#version

**Review by FabijanC — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mikiw) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (mikiw) [reply README.md]: done

**Pull request merged**